cmake/openblas.pc.in

```diff
@@ -7,5 +7,5 @@ Name: OpenBLAS
 Description: OpenBLAS is an optimized BLAS library based on GotoBLAS2 1.13 BSD version
 Version: @OPENBLAS_VERSION@
 URL: https://github.com/xianyi/OpenBLAS
-Libs: -L${libdir} -lopenblas${libsuffix}
+Libs: @OpenMP_C_FLAGS@ -L${libdir} -lopenblas${libsuffix} 
```

Comment:
Shouldn't these have been added one line lower, to the Cflags?

Reply:
Admittedly I was experimenting with the various CMAKE built-ins and did not switch to the cflags line after finding that the actual "library" variables did not return anything fit for direct insertion.  However I found examples of both and having it in Libs (or Libs.private) seemed more common. Presumably both will work ?